tests/test_visualizations.py

```diff
@@ -13,20 +13,29 @@
 # limitations under the License.
 
 import numpy as np
-import pandas as pd
 import pytest
 from qref import SchemaV1
 
 from bartiq import Routine, compile_routine
 from bartiq.symbolics.sympy_backend import SympyBackend
-from bartiq.visualizations import TreeMap, _dataframe_with_unique_routine_names
 
+try:
+    import pandas as pd
 
+    from bartiq.visualizations import TreeMap, _dataframe_with_unique_routine_names
+
+    SKIP_VIS_TESTS = False
```

Comment:
[comment] I'd personally go with `viz`, as that's more how you spell it. I'm not asking to change it, but actually, curious what are your thoughts on this ;)